AlfrescoApp/Supporting Files/en.lproj/Localizable.strings

```diff
@@ -674,13 +674,13 @@
  */
 "settings.title" = "Settings";
 "settings.general.title" = "General";
-"settings.sync.on.cellular" = "Cellular Sync";
-"settings.sync.on.cellular.description" = "Switch off to only sync favorite content when you have a Wi-Fi connection.";
+"settings.sync.on.cellular" = "Synced Content";
```

Comment:
Yes, I did wonder if the title should be longer, something like "Use Data for Sync"?

The previous title wasn't really clear.